src/platform/user/profile/vet360/containers/TimedAlertBox.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+class TimedAlertBox extends React.Component {
+  state = {
+    isVisible: this.props.startTimer,
+  };
+
+  componentDidMount() {
+    if (this.props.startTimer) {
+      if (this.intervalId === undefined) {
+        this.intervalId = setInterval(this.clearAlert, this.props.interval);
```

Comment:
should a default value for the `interval` prop be set?

Reply:
also, just for the sake of semantics, could you use `setTimeout` instead since this is only going to call the callback once